samples/java-spring-eventsourced-customer-registry/src/main/java/customer/api/CustomerEntity.java

```diff
@@ -19,17 +19,23 @@
 public class CustomerEntity extends EventSourcedEntity<Customer, CustomerEvent> {
   private static final Logger logger = LoggerFactory.getLogger(CustomerEntity.class);
 
+  record Confirm(String msg){
+    public static Confirm done = new Confirm("done");
+  }
+
   @GetMapping
   public Effect<Customer> getCustomer() {
     return effects().reply(currentState());
   }
 
   @PostMapping("/create")
-  public Effect<String> create(@RequestBody Customer customer) {
+  public Effect<Confirm> create(@RequestBody Customer customer) {
     logger.info("Creating {}", customer);
+
+    logger.info("what {}", Confirm.done);
```

Comment:
leftover?

Reply:
At least I didn't push the whole sentence. What that f...?